.env.example.qubex

```
@@ -48,6 +48,7 @@ SLACK_BOT_TOKEN=
 SLACK_CHANNEL_ID=
 SLACK_APP_TOKEN=
 ENABLE_LOCAL_CODEX_AGENT=false
+ENABLE_AGENT_CALIBRATION=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ENABLE_AGENT_CALIBRATION key should go before the ENABLE_LOCAL_CODEX_AGENT key

(UnorderedKey)